doc/glossary.rst

```diff
@@ -46,6 +46,20 @@ MNE-Python core terminology and general concepts
         :func:`mne.bem.make_watershed_bem` or :func:`mne.bem.make_flash_bem`.
         See :ref:`tut-forward` for usage demo.
 
+    dipole
+        See 'equivalent current dipole'.
```

Comment:
to be really thorough, this should be
```suggestion
        See :term:`equivalent current dipole`.
```
I realize they're adjacent, but in theory new terms could be added between them.

Reply:
that's just because I don't speak rst ;)